lib/workers/repository/update/pr/body/index.ts

```diff
@@ -32,22 +32,20 @@ function massageUpdateMetadata(config: BranchConfig): void {
       depNameLinked = `[${depNameLinked}](${primaryLink})`;
     }
 
-    let sourceRootPath = 'tree';
+    let sourceRootPath = 'tree/HEAD';
     if (sourceUrl) {
       const sourcePlatform = detectPlatform(sourceUrl);
       if (sourcePlatform === 'bitbucket') {
-        sourceRootPath = 'src';
+        sourceRootPath = 'src/HEAD';
```

Comment:
Is this a fix for the existing bitbucket cloud functionality?

Reply:
No, this doesn't change existing behavior. `src/HEAD` is already used with bitbucket but `HEAD` is currently hard-coded a few lines later:
https://github.com/renovatebot/renovate/blob/c4242426bcf43e0bfa5eaeb3fcceb4a0d5edda6a/lib/workers/repository/update/pr/body/index.ts#L48

All other platforms use `tree/HEAD`